src/material-experimental/mdc-input/input.spec.ts

```diff
@@ -1094,6 +1094,23 @@ describe('MatMdcInput with forms', () => {
     expect(formField._control.empty).toBe(false);
   }));
 
+  it('should update notch size after changing appearance to outline', fakeAsync(() => {
+    const fixture = createComponent(MatInputWithAppearance);
+    fixture.detectChanges();
+
+    expect(document.querySelector('.mdc-notched-outline__notch')).toBe(null);
```

Comment:
It might be better to use `fixture.nativeElement.querySelector`. Querying it off the document could fail if we have some other test that didn't clean up after itself. The same goes for the other two `querySelector` calls below.

Reply:
Good to know - I wondered why we might not be using `document` and that makes sense